docs/dqx/docs/reference/quality_rules.mdx

```diff
@@ -2234,6 +2234,118 @@ Using DQX classes:
 
 When using dataset-level checks, the filter condition is applied before aggregation, ensuring that the check operates only on the relevant subset of rows rather than on the aggregated results.
 
+## Converting checks between formats
+
+Checks in DQX can be defined using Python classes or YAML declarations. YAML files are first parsed into dictionaries and then converted into DQX classes behind the scenes. Because of this shared structure, all formats are interchangeable and can be safely converted between each other.
```

Comment:
```suggestion
In DQX, checks can be defined either as Python classes or YAML declarations. When using YAML, the files are first parsed into dictionaries and then transformed into DQX class instances under the hood. Since both formats share the same internal structure, they are fully interchangeable and can be safely converted between one another.
```